node/caching-compiler.js

```diff
@@ -48,7 +48,7 @@ const fallbackPkgInfo = {
   cache: Object.create(null)
 };
 
-function readWithCache(info, content) {
+function readWithCache(info, content, options) {
   const json = info && info.json;
   const reify = json && json.reify;
   const cacheFilename = getCacheFilename(content, reify);
```

Comment:
But the options might affect the compiler output, and we want `getCacheFilename` to return different names for different versions of the compiled code.